data/brands/amenity/recycling.json

```diff
@@ -107,6 +107,21 @@
         "name": "Ecopunto"
       }
     },
+    {
+      "displayName": "Islamic Relief",
+      "locationSet": {"include": ["gb"]},
```

Comment:
And I was confused by this locationSet because wikipedia give a map
it shows that Islamic Relief is operating world wide.

https://en.wikipedia.org/wiki/Islamic_Relief#Islamic_Relief_offices_and_partners

<p><a href="https://commons.wikimedia.org/wiki/File:WorldMap-Partner%26FieldOffices-AR2020.jpg#/media/File:WorldMap-Partner&amp;FieldOffices-AR2020.jpg"><img src="https://upload.wikimedia.org/wikipedia/commons/b/b4/WorldMap-Partner%26FieldOffices-AR2020.jpg" alt="A 2020 map of where Islamic Relief operates" width="1754" height="1240"></a><br>By &lt;a href="//commons.wikimedia.org/w/index.php?title=User:Mojokola&amp;amp;action=edit&amp;amp;redlink=1" class="new" title="User:Mojokola (page does not exist)"&gt;Mojokola&lt;/a&gt; - &lt;span class="int-own-work" lang="en"&gt;Own work&lt;/span&gt;, <a href="https://creativecommons.org/licenses/by-sa/4.0" title="Creative Commons Attribution-Share Alike 4.0">CC BY-SA 4.0</a>, <a href="https://commons.wikimedia.org/w/index.php?curid=110628844">Link</a></p>

Reply:
They have a global presence but based on my knowledge and searching online I can't find any information suggesting they have clothing banks outside the UK